Sources_v3/APIClient/Endpoints/ChannelEndpoints.swift

```diff
@@ -107,4 +107,14 @@ extension Endpoint {
             body: ["remove_members": userIds]
         )
     }
+    
+    static func event<ExtraData: ExtraDataTypes>(cid: ChannelId, eventType: EventType) -> Endpoint<EventPayload<ExtraData>> {
```

Comment:
`sendEvent`?

Reply:
I think the response type can be `EmptyResponse`

Reply:
This needs to be tested

Reply:
I want to keep `EventPayload<ExtraData>` to make sure the response was decoded successfully, but the result for user side less important.